utils/storybook/withSnapshotsEnabled.ts

```diff
@@ -0,0 +1,55 @@
+type StoryDetails = {
+  title: string;
+  component?: React.ReactNode;
+  decorators?: any[];
+  parameters?: {
+    [key: string]: any;
+  };
+};
+
+type StoryFn = {
+  (): JSX.Element;
+  story?: {
+    parameters?: {
+      chromatic?: {};
+    };
+  };
+};
+
+/**
+ * A function to enable chromatic snapshots for a single story or a set of stories.
+ * To apply to all stories in a file, wrap the default export of a CSF story file.
+ * To apply to a single story, wrap the story fn.
+ *
+ * TODO: Clean up after optional chaining is enabled
+ */
+export const withSnapshotsEnabled: {
```

Comment:
I figured you would just rebase on #688 rather than pulling its files in. Do you mind if we hold off on merging this until that goes in so that these files are associated with that PR instead of this one?

Reply:
that works for me